docker-compose.yml

```diff
@@ -166,6 +166,32 @@ services:
     restart: always
     labels:
       - "logging=true"
+  gn-publications-consumer:
+    image: lblod/delta-consumer:0.0.24
+    environment:
+      DCR_SYNC_BASE_URL: "https://gn-harvester.s.redhost.be/"
+      DCR_SERVICE_NAME: "besluiten-consumer"
+      DCR_SYNC_FILES_PATH: "/sync/besluiten/files"
+      DCR_SYNC_DATASET_SUBJECT: "http://data.lblod.info/datasets/delta-producer/dumps/lblod-harvester/BesluitenCacheGraphDump"
+      DCR_INITIAL_SYNC_JOB_OPERATION: "http://redpencil.data.gift/id/jobs/concept/JobOperation/deltas/consumer/initialSync/besluiten"
+      DCR_DELTA_SYNC_JOB_OPERATION: "http://redpencil.data.gift/id/jobs/concept/JobOperation/deltas/consumer/besluitenSyncing"
+      DCR_JOB_CREATOR_URI: "http://data.lblod.info/services/id/besluiten-consumer"
+      DCR_KEEP_DELTA_FILES: "false"
+      DCR_DELTA_FILE_FOLDER: "/consumer-files"
+      DCR_DISABLE_DELTA_INGEST: "true" # toggle this in override, leave 'true' in default docker-compose.yml
+      DCR_DISABLE_INITIAL_SYNC: "true" # toggle this in override, leave 'true' in default docker-compose.yml
+      BYPASS_MU_AUTH_FOR_EXPENSIVE_QUERIES: "false"
+      DIRECT_DATABASE_ENDPOINT: "http://virtuoso:8890/sparql"
+      BATCH_SIZE: 250
```

Comment:
From lokaalbeslist, I think we can tweak a bit:
```
      BATCH_SIZE: "1"
      SLEEP_BETWEEN_BATCHES: "1"
      PARALLEL_CALLS: "4"
```

Reply:
Do you mean adding these in `docker-compose.override.yml`?

Reply:
yes